packages/operation/src/evm/deploy/account-assignment.ts

```diff
@@ -55,8 +55,8 @@ export async function assignDesignatedWallets(state: State) {
       };
     });
 
-    requestersById[configRequester.id] = { ...requester, designatedWallets };
+    sponsorsById[configSponsor.id] = { ...sponsor, designatedWallets };
   }
 
-  return { ...state, requestersById };
+  return { ...state, requestersById: sponsorsById };
```

Comment:
Shouldn't this also be `sponsorsById`?
```suggestion
  return { ...state, sponsorsById };
```

Reply:
We should try set explicity types so that TS catches things like this

Reply:
Excelent catch! and I completely agree we should have explicit types.

Reply:
Allthough in this particular explicit types might not have caught it because we did have a requestersById object